docs/reference/model_configuration.md

```diff
@@ -10,7 +10,7 @@ Configuration options for SQLMesh model properties. Supported by all model kinds
 
 | Option                | Description                                                                                                                                                                                                                                                                                      |       Type        | Required |
 | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :---------------: | :------: |
-| `name`                | The model name. Must include at least a qualifying schema (`<schema>.<model>`) and may include a catalog (`<catalog>.<schema>.<model>`).                                                                                                                                                         |        str        |    Y     |
+| `name`                | The model name. Must include at least a qualifying schema (`<schema>.<model>`) and may include a catalog (`<catalog>.<schema>.<model>`).                                                                                                                                                         |        str        |    N     |
```

Comment:
Update this as well to shortly mention the name inference option (perhaps link the relevant section?)